src/browser/components/SavedScripts/SavedScriptsFolder.tsx

```diff
@@ -116,6 +119,24 @@ function SavedScriptsFolder({
       >
         Delete folder
       </ContextMenuItem>
+    ),
+    exportScripts && (
+      <ContextMenuItem
+        data-testid="contextMenuExport"
+        onClick={() => exportScripts('ZIPFILE')}
+        key="exportZip"
+      >
+        Export scripts as .zip file
+      </ContextMenuItem>
+    ),
+    exportScripts && (
+      <ContextMenuItem
+        data-testid="contextMenuRemove"
```

Comment:
looks like a wrong name here? `Remove` should be `Export` or some variant of that?